app/test_main.py

```diff
@@ -1,3 +1,53 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_data",
+    [
+        (
+            0,
+            0,
+            [0, 0]
+        ),
+        (
+            14,
+            14,
+            [0, 0]
+        ),
+        (
+            15,
+            15,
+            [1, 1]
+        ),
+        (
+            23,
+            23,
+            [1, 1]
+        ),
+        (
+            24,
+            24,
+            [2, 2]
+        ),
+        (
+            27,
+            27,
+            [2, 2]
+        ),
+        (
+            28,
+            28,
+            [3, 2]
+        ),
+        (
+            100,
+            100,
+            [21, 17]
+        ),
+    ]
```

Comment:
The test cases do not cover edge cases such as negative numbers or extremely large numbers. According to the checklist, you should ensure that your tests check all edge situations, such as if the function receives data out of the normal range, like negative numbers or very large numbers.